mesheryctl.sh

```diff
@@ -16,21 +16,35 @@ adapters["traefik_mesh"]=meshery-traefik-mesh:10006
 main() {
 	local perf_filename=
 	local perf_profile_name=
+	local endpoint_url=
 
 	parse_command_line "$@"
 
 	# perform the test given in the provided profile_id
 	if [ -z "$perf_profile_name" ]
 	then
+		for mesh in "${!adapters[@]}"
+		do
+			shortName=$(echo ${adapters["$mesh"]} | cut -d '-' -f2 | cut -d ':' -f1)
 
-		mesheryctl perf apply --file $GITHUB_WORKSPACE/.github/$perf_filename -t ~/auth.json
+			docker network connect bridge meshery_meshery-"$shortName"_1
+			docker network connect minikube meshery_meshery-"$shortName"_1
+		done
+
+		docker network connect bridge meshery_meshery_1
```

Comment:
We're not deploying in cluster so as to avoid burdening the cluster with load, yeah?

Reply:
This was a maybe in my mind but I thought it wouldn't matter but I guess it does. (It is not a bug, it is a feature 😅 )

But @DelusionalOptimist had added in-cluster deployments in a separate branch so I kind of held off adding it here since we might merge the branches. And the workflow needs to be only run in either of these so since we already have one way, we can start with it.